### PR TITLE
add environ variable

### PR DESCRIPTION
**Pull request by OkonSamuel**

### CODE REVIEW

**Comment by ablaom** [comment]
@OkonSamuel Go ahead and turn "Draft" off when you're ready for me to have a look.

**Comment by OkonSamuel** [comment]
Okay

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: CI
+
 env:
   PYTHON: Conda
+  LD_LIBRARY_PATH: /home/runner/.julia/conda/3/lib
```
Comment (ablaom) [inline .github/workflows/ci.yml:6]:
```suggestion
# remove next line (and others marked below) when #42 properly resolved
  LD_LIBRARY_PATH: /home/runner/.julia/conda/3/lib
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -42,6 +45,10 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@v1
+      - name: "Install Conda"
```
Comment (ablaom) [inline .github/workflows/ci.yml:50]:
```suggestion
# remove next four lines (and one other marked above) when #42 properly resolved
      - name: "Install Conda"
```

**Review by ablaom — APPROVED** [review]
Great work, @OkonSamuel, very much appreciated.

**Pull request merged**